cfme/tests/v2v/test_rest_migrations.py

```diff
@@ -0,0 +1,126 @@
+"""Test to check v2v migration rest API"""
+import fauxfactory
+import pytest
+
+from cfme.infrastructure.provider.rhevm import RHEVMProvider
+from cfme.infrastructure.provider.virtualcenter import VMwareProvider
+from cfme.markers.env_markers.provider import ONE_PER_VERSION, ONE_PER_TYPE
+
+
+pytestmark = [
+    pytest.mark.provider(
+        classes=[RHEVMProvider],
+        selector=ONE_PER_VERSION,
+        required_flags=["v2v"]
+    ),
+    pytest.mark.provider(
+        classes=[VMwareProvider],
+        selector=ONE_PER_TYPE,
+        fixture_name="second_provider",
+        required_flags=["v2v"]
+    )
+]
+
+
+@pytest.fixture(scope="function")
+def get_clusters(appliance, v2v_providers):
+    clusters = {}
+    source_cluster = v2v_providers.vmware_provider.data.get("clusters")[0]
```

Comment:
Please wrap these in a `try/except` block to catch the IndexError and call `pytest.skip` instead of an exception raising to the result.

Reply:
sure, updating other indexing too. :+1: